lm_eval/tasks/benczechmark/benczechmark_snli.yaml

```diff
@@ -0,0 +1,94 @@
+group: benczechmark_snli
+group_alias: CS SNLI
+include:
+ - templates/metrics/multichoice_metrics_yaml
+fewshot_split: test
+task:
+  - task: snli
+    task_alias: prompt-0
+    include: templates/multichoice/multi_class_template_yaml
+    dataset_path: CZLC/cs_snli
+    description: "Budu psát dvě věty, kterým říkám premisa a hypotéza. Hypotéza může z premisy vyplývat, nebo jí může protiřečit. Pokud ani nevyplývá, ani neprotiřečí, je neutrální. Vyhodnoťte, zda hypotézy vyplývají z premis, nebo jim protiřečí, nebo jsou neutrální."
+    doc_to_target: gold
```

Comment:
If you want to use content of "gold" attribute as a target index, then it needs to be present in your dataset.
I checked it on CZLC and it is not present there now. Its contents need to be only numbers 0,1, or 2, corresponding to the index of the correct class.
There is how one example in your jsonl looks now.

```
{"cze_hypothesis":"Muž se usmívá na svou ženu.","cze_premise":"Muž v černé košili se usmívá na ženu v černé košili, jaké tetování a kroužek na obočí.","eng_hypothesis":"A man is smiling at his wife.","eng_premise":"A man in a black shirt is smiling at a woman in a black shirt what a tattoo and a eye brow ring.","id":"CZdevSNLI_298","label_cze":"neutral","label_eng":"neutral","label_cze1":"","dataset":"CZdevSNLI","row_num":298}

```
Martin

Reply:
You can also define label_cze as the target field, and convert its value into index with some jinja2 magic.
```
{{ 0 if label == 'entailment' else 1 if label == 'contradiction' else 2 if label == 'neutral' else -1 }}
```

Reply:
thx, stupid error, really. now, acc=0.4